web/packages/teleport/src/services/joinToken/joinToken.ts

```diff
@@ -73,7 +73,7 @@ class JoinTokenService {
   fetchJoinTokens(signal: AbortSignal = null): Promise<{ items: JoinToken[] }> {
     return api.get(cfg.getJoinTokensUrl(), signal).then(resp => {
       return {
-        items: resp.items.map(makeJoinToken),
+        items: resp.items?.map(makeJoinToken) || [],
```

Comment:
Does the API endpoint not include `items` when there are no items to return?

Reply:
It includes it, but it's `null`:
<img width="1557" alt="image" src="https://github.com/user-attachments/assets/d371b301-ebd7-4ba5-9719-6251369b0c2d">

Reply:
Looks like it has to do with [not using zero-length slices](https://stackoverflow.com/questions/56200925/return-an-empty-array-instead-of-null-with-golang-for-json-return-with-gin).